analysis/src/CompletionJsx.ml

```diff
@@ -559,6 +559,19 @@ let getJsxLabels ~componentPath ~findTypeOfValue ~package =
              && Path.last path = "props" ->
         (* JSX V4 external or interface *)
         getFieldsV4 ~path ~typeArgs
+      | Tarrow (Nolabel, typ, _, _) -> (
+        (* Component without the JSX PPX, like a make fn taking a hand-written
```

Comment:
Should one check anything about the return type? Or can it be anything?

Reply:
We could check the return type for sure but I'm unsure if it really matters that much. Strictly though it needs to be `React.element`, or whatever JSX element is configured (not sure if it can be configured now, but it'll be possible later). For this to error, you'd need to use a JSX component with a make function in a module that takes a type named props, and return something else then React.element (which will error in compilation anyway).

I say we go without it for this case, and then fix it if need be?

Reply:
Sounds good